Bubbly UI

tokaa1 opened the pull request:
[Make PearAI look more bubbly](https://github.com/orgs/trypear/projects/20/views/1?pane=issue&itemId=100908299&issue=trypear%7Cpearai-app%7C212)
I made PearAI more bubbly 🤪🤪🤪

(Dark Modern)
![Screenshot 2025-04-15 at 9 22 01 PM](https://github.com/user-attachments/assets/618c530d-2959-4e76-a2d5-64e5bfe75d58)
(Snazzy Light)
![Screenshot 2025-04-15 at 9 23 19 PM](https://github.com/user-attachments/assets/e5a13997-57a6-45b4-a25d-60c77718af95)

I made UI close to the mock, except for Explorer which is a kinda different. Haven't seen any bugs yet pray

Lmk what you think about the design and implementation.

Most changes are in a 'bubbly.css' where they add onto or sometimes modify existing styles. This was just easiest at the time, and I'm not sure if it would be better for the codebase to keep these styles separate in bubbly.css or not.

Lets gooooooooooooooo


<!-- ELLIPSIS_HIDDEN -->

----

> [!IMPORTANT]
> Introduces a new 'bubbly' UI style with rounded elements and updated layout logic for PearAI.
> 
>   - **UI Styling**:
>     - Introduces `bubbly.css` for new rounded UI styles, affecting border-radius and background colors.
>     - Applies `bubbly-part` class to `Part` in `part.ts` and `EditorPart` in `editorPart.ts` for rounded styles.
>     - Updates `editorTabsControl.ts` to adjust tab height and padding for bubbly appearance.
>   - **Layout Adjustments**:
>     - Modifies `PartLayout` in `part.ts` to reduce dimensions for bubbly parts.
>     - Updates `layoutService.ts` to include `bubblyParts` set for identifying bubbly components.
>   - **Misc**:
>     - Imports `bubbly.css` in `style.ts` for global application of new styles.
> 
> <sup>This description was created by </sup>[<img alt="Ellipsis" src="https://img.shields.io/badge/Ellipsis-blue?color=175173">](https://www.ellipsis.dev?ref=trypear%2Fpearai-app&utm_source=github&utm_medium=referral)<sup> for 1754988b8b97ac63d2a639bbcf64ba11c2ed97dd. It will automatically update as commits are pushed.</sup>

<!-- ELLIPSIS_HIDDEN -->

0xleonz commented:
mmmm

Fryingpannn commented:
Nicee

tokaa1 committed:
Bubbly UI 🫧🫧

tokaa1 committed:
Make panel same color as editor

tokaa1 committed:
Detached activity bar 🔥🔥🔥

tokaa1 committed:
Remove border from sidebar activity bar

tokaa1 committed:
All monaco-inputbox is round

tokaa1 committed:
Remove borders from some of the non-bubbly parts

tokaa1 committed:
Typo - round parts by default

tokaa1 committed:
Lower tooltip hover delay

tokaa1 committed:
Final touches + improved bubbly system

tokaa1 commented:
![Screenshot 2025-04-16 at 4 40 09 PM](https://github.com/user-attachments/assets/fd1a3c98-bc98-4532-a1a0-ac96a7b98020)

tokaa1 committed:
Border radius consistency + small touches. Also decluttered editor tabs

tokaa1 committed:
MUCH PROGRESS (sidebar, activitybar, consistency)

nang-dev commented on src/vs/workbench/browser/media/bubbly.css:
```diff
@@ -0,0 +1,156 @@
+
+.bubbly-part .tab-border-top-container {
+	height: 0 !important;
+	background-color: none !important;
```
@tokaa1 @Himanshu-Singh-Chauhan is this tru

tokaa1 replied:
Man somehow i guess

tokaa1 committed:
Clean up css and bug fix

Himanshu-Singh-Chauhan committed:
blahhh

nang-dev committed:
Bumped to v1.8.8

nang-dev committed:
Bumped to v1.8.9

tokaa1 committed:
fix editor tabs when using wrap tabs

tokaa1 committed:
Round out notfications

nang-dev commented:
WE ARE ALMOST THERE!

here's the QA:

1. 

<img width="460" alt="image" src="https://github.com/user-attachments/assets/0e577107-9b3b-42ed-b275-69c136a061e1" />

2. 

<img width="490" alt="image" src="https://github.com/user-attachments/assets/6a15bcac-d7a9-4f64-8a42-7d054757cb57" />

3. 
<img width="372" alt="image" src="https://github.com/user-attachments/assets/71e1dd71-d3e1-407f-8b10-67c6eb9db055" />

4. if this is not possible, then lets just make it not bubbly
<img width="468" alt="image" src="https://github.com/user-attachments/assets/8db40b05-d52a-4f1c-bd00-b54c05912036" />

5. 
<img width="398" alt="image" src="https://github.com/user-attachments/assets/d07f2d5e-86cc-48f5-b83f-726baf25d52e" />

6. whats up with the icons here?
<img width="480" alt="image" src="https://github.com/user-attachments/assets/afc7e660-f369-4028-b0e2-8cdc3daf049c" />

tokaa1 committed:
Make text transforms uppercase again

tokaa1 committed:
consistent panel styling + other spacing nits 🚀

tokaa1 committed:
even padding on right side of panel

tokaa1 committed:
sizings in bottom panel + tiny nits

tokaa1 commented:
Fixed all nits!
Anything not in notes was updated identical to above

### Update notes:
Colors for icons were wrong before because the activity bar background was supposed to be different (because of theme, Light+ in screenshots).
![Screenshot 2025-05-02 at 5 16 06 PM](https://github.com/user-attachments/assets/a5b7f1f1-358f-437e-b751-6273db061616)
What it looks like now:
![Screenshot 2025-05-02 at 5 17 04 PM](https://github.com/user-attachments/assets/855886cc-4b30-4e65-9d3b-9dee1021437f)


Updated the bottom panel for spacing nits and other stuff
(was not able to make it look like this, so that spot is not bubbly)
![Screenshot 2025-05-02 at 5 22 45 PM](https://github.com/user-attachments/assets/2b8f4e51-61d4-4030-8e3a-6dc85cee0d2f)
Now looks like this:
![Screenshot 2025-05-02 at 5 19 00 PM](https://github.com/user-attachments/assets/80be1314-d6eb-42d2-a20a-ba8b45d2dbcc)

🔥🔥
